.eslintrc.json

```diff
@@ -4,19 +4,28 @@
         "commonjs": true,
         "es2021": true
     },
-    "extends": ["eslint:recommended", "airbnb-base"],
+    "extends": [
+        "eslint:recommended",
+        "airbnb-base",
+        "airbnb-typescript/base",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:import/typescript",
+        "prettier"
```

Comment:
Is this configuring eslint with prettier properly? If so there is an issue I created that can be grouped into this PR if it isn't already.